src/vm/methodtable.h

```diff
@@ -390,6 +390,12 @@ struct MethodTableWriteableData
 
         enum_flag_SkipWinRTOverride         = 0x00000100,     // No WinRT override is needed
 
+        enum_flag_HasOverriddenEquals       = 0x00000200,     // Is any field type or sub field type overrode Equals
+        enum_flag_HasCheckedOverriddenEquals   = 0x00000400,  // Whether we have checked the overridden Equals
+
+        enum_flag_HasOverriddenGetHashCode  = 0x00000800,     // Is any field type or sub field type overrode GetHashCode
```

Comment:
Equals and GetHashCode are likely going to be used together. I think you can check for both at the same time. It will save the bit here, and it will be a bit faster for the typically case.